packages/apps/graph/generated-schema.graphql

```diff
@@ -340,9 +340,9 @@ type Query {
 
   """
   Retrieve events by qualifiedName (e.g. `coin.TRANSFER`). Default page size is 20.
-         
```

Comment:
These should not be removed. They are required to have new lines in the GraphiQL docs, since regular newlines make the rest of the text dissapear.